rsc/src/main/java/com/cloudera/livy/rsc/RSCConf.java

```diff
@@ -75,7 +75,9 @@
     TEST_STUCK_START_DRIVER("test.do_not_use.stuck_start_driver", false),
 
     JOB_CANCEL_TRIGGER_INTERVAL("job_cancel.trigger_interval", "100ms"),
-    JOB_CANCEL_TIMEOUT("job_cancel.timeout", "30s");
+    JOB_CANCEL_TIMEOUT("job_cancel.timeout", "30s"),
+
+    RETAINED_STATEMENT_NUMBER("retained_statements", 100);
```

Comment:
@tc0312 is it enough to set 100 as a default value, or should we increase the value like 1000?